app/components/sign-up.jsx

```diff
@@ -0,0 +1,325 @@
+                    <input
+                        name='last-name'
+                        className={cx(classes.input, isLogIn && classes.disabled)}
+                    />
+                </div>
+                <div className={cx(classes.inputBoxes, !state.email && isSubmitted && classes.invalid)}>
+                    <p id="text">E-mail</p>
+                    <input
+                        name='email'
+                        className={cx(classes.input, !state.email && isSubmitted && classes.invalidInput)}
+                        onChange={e => methods.onChangeEmail(e.target.value)}
+                    />
+                </div>
+                <div className={cx(classes.inputBoxes, !state.password && isSubmitted && classes.invalid)}>
+                    <p id="text">Password</p>
+                    <input
+                        name='password'
+                        type='password'
+                        className={cx(classes.input, !state.password && isSubmitted && classes.invalidInput)}
+                        onChange={e => methods.onChangePassword(e.target.value)}
+                    />
+                </div>
+                <div className={cx(classes.inputBoxes, isLogIn && classes.disabled)}>
+                    <p id="text">Confirm Password</p>
+                    <input
+                        name='confirm'
+                        type='password'
+                        className={cx(classes.input, isLogIn && classes.disabled)}
+                        onChange={e => methods.onChangeConfirm(e.target.value)}
+                    />
+                </div>
+                <div className={cx(classes.agreeTermContainer, isLogIn && classes.disabled)}>
+                    <div className={classes.checkTerm}>
+                        <input
+                            className={classes.checkTermBox}
+                            type='checkbox'
+                            name='agreed'
+                            onClick={e => methods.onChangeAgree(e.target.checked)}
+                        />
+                        <label className={classes.agreeTermLabel}>
+                            Yes, I agree to the
+                            <a href='https://enciv.org/terms' target='_blank' className={classes.aLinkTerm}>
+                                Terms of Service
+                            </a>
+                            and
+                            <a href='https://enciv.org/privacy' target='_blank' className={classes.aLinkTerm}>
+                                Privacy Policy.
+                            </a>
+                        </label>
+                    </div>
+                </div>
+                <div className={cx(classes.error, !(state.error || state.info || state.success) && classes.disabled)}>
+                    <SvgAlertTriangle viewBox="0 0 24 20" />
+                    <span> Oops! </span>
+                    {state.error && <span>{state.error}</span>}
```

Comment:
Okay, sorry about the details.  IF the message is an error, render the red box with the Opps as is now.  If it's info, just render the text, in the default color, if it's success render it in bold green.  I won't ask the designer for icons for those things, lets just do this and get it done.   Also - it's okay to ask me if it looks like I'm telling you to do something dumb (like render oops on success).